tools/mkmodel/mkmodel.c

```diff
@@ -95,7 +122,6 @@ void print_args( char * name )
     fprintf(stderr, "Command-line flags:\n");
     fprintf(stderr, "   -o/--output <dir>       Specify output directory (default: .)\n");
     fprintf(stderr, "   --anim-no-stream        Disable animation streaming\n");
-    fprintf(stderr, "   --anim-fps <value>      Frame rate of the animation (default: 30.0)\n");
```

Comment:
Why is this getting removed?

Reply:
Because animation framerate is just a parameter of the reduction algorithm now. It is only used to generate the initial samples.